backend/tests/test_state_transitions.py

```diff
@@ -0,0 +1,70 @@
+from copy import deepcopy
+from datetime import datetime
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+from program.media import Show, Season, Episode, Movie
+from program.state_transition import process_event
+from program.libaries import SymlinkLibrary
+from program.settings.manager import settings_manager
+from program.media.state import States
+from program.indexers import TraktIndexer
+
+
+@pytest.fixture
+def test_show():
+    return Show({'imdb_id': 'tt1405406'})
+
+@pytest.fixture
+def test_library_items():
+    settings_manager.settings.symlink.library_path = Path(__file__).parent / "library"
+    library = SymlinkLibrary()
+    return [item for item in library.run()]
+
+def test_symlink_library_output(test_library_items):
+    assert all(i.state == States.Completed for i in test_library_items)
```

Comment:
I would stick to given, when, then format for writing tests. It gives the test structure and actual tells the reader whats going on. Now all im seeing is a bunch of asserts against a test fixture.